#152798637 Landing Page

pawnjester opened the pull request:
#### What does this PR do?
Users should see a landing page with effective UI/UX

#### Description of Task to be completed?
* write the html and css code for the landing page
* remove existing code in the index.html and replace with the html code written
* remove existing style sheet and replace with the custom css code written
* include the stylesheet in the `head.jade` file
* include `animate.css` for animation in `head.jade`

#### How should this be manually tested?
* run the command `mongod` to start the mongo server
* run `yarn start` to start the server
* access `localhost:3000` to view the landing page

#### Any background context you want to provide?
Nil
#### What are the relevant pivotal tracker stories?
#152798637
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2017-12-04 at 5 48 05 pm" src="https://user-images.githubusercontent.com/26750279/33569478-1f1340b4-d92a-11e7-9c8d-14d75697ea6f.png">

<img width="1440" alt="screen shot 2017-12-04 at 7 52 46 pm" src="https://user-images.githubusercontent.com/26750279/33570332-cd598e7e-d92c-11e7-87a2-a18f968c4019.png">

<img width="1435" alt="screen shot 2017-12-04 at 7 52 55 pm" src="https://user-images.githubusercontent.com/26750279/33570346-d99eea76-d92c-11e7-89db-156912cb1b5c.png">

<img width="1439" alt="screen shot 2017-12-04 at 7 53 02 pm" src="https://user-images.githubusercontent.com/26750279/33570360-e74ee766-d92c-11e7-9701-dd717a828a24.png">


#### Questions:
Nil

Commit:
feat(landing-page):landing-page
- edit landing page design

sachungo commented on app/views/includes/head.jade:
```diff
@@ -23,11 +23,23 @@ head
 
-  link(rel='stylesheet', href='/lib/bootstrap/dist/css/bootstrap.css')
-  link(rel='stylesheet', href='/css/common.css')
+  //- link(rel='stylesheet', href='/lib/bootstrap/dist/css/bootstrap.css')
```
Why not just remove it instead of commenting it out? Will it still be used in future?

pawnjester replied:
No, it would not be used in the future, I have removed it

sachungo commented on public/css/index.css:
```diff
@@ -0,0 +1,175 @@
+    min-height: 700px;
+    background-size: cover;
+    background-attachment: fixed;
+    /* opacity: 0.6; */
```
If not being used, then remove it

pawnjester replied:
done

sachungo commented on public/css/index.css:
```diff
@@ -0,0 +1,175 @@
+
+.img-div {
+    margin: 0 auto;
+}
```
Add new line
It might be easier to set up your editor to add it automatically when saving

pawnjester replied:
done

sachungo reviewed (changes requested):
I have tested it locally and the landing page looks good. 

I have left a few feedback to be addressed but they are nothing major.

I would suggest that the images be added to their own directories for ease of future reading. E.g. the team should be in a folder such as `public/img/team/<member.png>`

I have a few nitpicks:
- What should the text alignment on cards be? Centered or left-aligned?
- The text for the second card on `How to Play` is missing a word. It doesn't flow properly when reading it.
- Chuks has 2 images in the repo yet only one has been used. Delete the other one if it won't be used

Commit:
feat(implement-feedback): implement feeback
- create a folder for the card images
- add a new line in the index.css
- center the text in the card image
- remove unused code
[Fixes #152798637]

sachungo commented on app/views/includes/head.jade:
```diff
@@ -23,11 +23,20 @@ head
 
-  link(rel='stylesheet', href='/lib/bootstrap/dist/css/bootstrap.css')
-  link(rel='stylesheet', href='/css/common.css')
+
```
Remove the 1 extra line

sachungo commented on app/views/includes/head.jade:
```diff
@@ -23,11 +23,20 @@ head
+  link(rel='stylesheet', href='/css/index.css')
   link(rel='stylesheet', href='/css/style.css')
 
+  
```
Same as here

sachungo commented on app/views/includes/head.jade:
```diff
@@ -23,11 +23,20 @@ head
+  crossorigin="anonymous")
+  script(src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb"
+      crossorigin="anonymous")
+
```
Same as here

sachungo reviewed (commented):
Left minor comments regarding empty spaces.

Also, ensure the indentation in `index.html` space is 2 spaces instead of 4, when using tab

Commit:
feat(implement-feedback):implement feedback
- change to indentation from 4 spaces to 2
- change card image

sachungo commented on public/views/index.html:
```diff
@@ -1,142 +1,136 @@
-        <span class="navbar-toggler-icon"></span>
+    <a class="navbar-brand type" href="#" style="cursor:pointer">
+      <span id="cards">Cards</span> For Humanity</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
```
If an opening tag has too many attributes, then you can break it down as:
```html
<button
  class="navbar-toggler"
  type="button"
  data-toggle="collapse"
  data-target="#navbarSupportedContent"
  aria-controls="navbarSupportedContent"
  aria-expanded="false"
  aria-label="Toggle navigation"
>

Commit:
feat(implement-feedback):implement feedback
- break down button tag

Commit:
feat(implement-feedback):implement feedback
- break down button tag

Commit:
feat(implement-feedback):implement feedback
- indent attributes in button

sachungo reviewed (approved):
All feedback have been addressed
